src/modules/map/components/toolbox/TimeSlider.vue

```diff
@@ -174,20 +211,20 @@ onMounted(() => {
         trigger: 'manual',
         theme: 'danger',
     })
+    tippyInstance.hide()
 })
 
 onUnmounted(() => {
-    if (!isPristine.value) {
-        activeLayers.value.forEach((layer, index) => {
-            if (layer.hasMultipleTimestamps) {
-                store.dispatch('setTimedLayerCurrentYear', {
-                    index,
-                    year: previewYear.value,
-                    ...dispatcher,
-                })
-            }
-        })
-    }
+    activeLayers.value.forEach((layer, index) => {
+        if (layer.hasMultipleTimestamps) {
+            store.dispatch('setTimedLayerCurrentYear', {
+                index,
+                year: previewYear.value,
+                ...dispatcher,
+            })
+        }
```

Comment:
This was probably already a bug before, but here you cannot add the previewYear to all layer, as you might have layer with invalid timestamp, like in this example https://sys-s.dev.bgdi.ch/rdutl0i1t7wz

Reply:
Fixed it, it doesn't update the layers that won't have the year available. I can't push `null` at this point, this would need further development.